### ISSUE TITLE
Missing tooltip for split button's down arrow

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#11833]
## 📝 Provide detailed reproduction steps (if any)

1.  Open document editor example.
2.  Hover the pointer over down arrow of the bulleted list button.

### ✔️ Expected result

It should show a tooltip like it does for the button's face. It helps to understand the purpose of a button.

![](https://user-images.githubusercontent.com/5353898/170682689-92e27ba2-7d80-45b2-bc43-2c61f145a1a8.png)

### ❌ Actual result

The tooltip is missing.

![](https://user-images.githubusercontent.com/5353898/170682541-4c5f9e92-06e8-429d-814b-64b19a52d7df.png)

## 📃 Other details

The question is how should the tooltip act on it?
* Should a separate tooltip (with the same text) be applied to both parts of the split button?
* Should the tooltip be shared by button's face and the down arrow icon?

Gdocs implement a separate tooltip for both.

*   Browser: Any
*   OS: Any
*   First affected CKEditor version: All

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
The question is how should the tooltip act on it?

*   Should a separate tooltip (with the same text) be applied to both parts of the split button?
*   Should the tooltip be shared by button's face and the down arrow icon?

I did a quick research on this:

<table><tbody><tr><td>&nbsp;</td><td>Hovering/focusing button face</td><td>Hovering/focusing arrow part</td></tr><tr><td>MS Word</td><td><figure class="image"><img src="https://user-images.githubusercontent.com/5353898/171021492-661b789c-d69a-40bb-b886-c35e15f0a63b.png"></figure></td><td><figure class="image"><img src="https://user-images.githubusercontent.com/5353898/171021459-796f8d81-2cc3-4ef8-a87f-9ede9865624d.png"></figure></td></tr><tr><td>GDocs</td><td><figure class="image"><img src="https://user-images.githubusercontent.com/5353898/171021908-9d534bc9-b972-424c-a4cf-02633e3846fe.png"></figure></td><td><figure class="image"><img src="https://user-images.githubusercontent.com/5353898/171021914-a2f9eba5-c73b-403a-81cf-1075c2576de6.png"></figure></td></tr><tr><td>Other WYSIWYGs</td><td><figure class="image"><img src="https://user-images.githubusercontent.com/5353898/171022367-438d5648-fb85-4515-9bf0-ffa9284bd84b.png"></figure></td><td><figure class="image"><img src="https://user-images.githubusercontent.com/5353898/171022159-0021fbbb-57c9-4a0d-9c2e-5f7827cb469c.png"></figure></td></tr></tbody></table>

It's safe to say that most commonly we see a separate label for button face and the dropdown arrow. I also feel like it will be the list amount of work for us - so let's have it implemented this way.
